### ISSUE TITLE
Carousel indicators not working

### ISSUE THREAD

**Issue by hotrush** [yuche/vue-strap#131]
Seems that carousel indicators not working. It creates an empty ol element, but no list items added

![http://dl1.joxi.net/drive/0010/1661/693885/160302/efe554c292.png](http://dl1.joxi.net/drive/0010/1661/693885/160302/efe554c292.png)

demo also not working. can anybody help?

**Comment by unematiii** [comment]
1) `vue-strap.js` has a type on line 834: the `<li>` tag is unclosed.
2) Depending on your Vue.js version
  - < 1.0, `v-for` & `v-bind` directives are not supprted (line 834)
  - > 1.0 `inherit: true` is deperecated (line 833), indicator component cannot access parent vars.

Currently trying to fix for Vue > 1.0 .

**Comment by hotrush** [comment]
You are right, i'm using 1.0+ version, and i have fixed that all in my fork here https://github.com/hotrush/vue-strap/blob/master/src/Slider.vue and herehttps://github.com/hotrush/vue-strap/blob/master/src/Carousel.vue.
But i'm using custom templates, so cannot update a pull request

**Comment by unematiii** [comment]
Actually this is the case for me also, I needed to add embedded video player.

**Comment by FerchoCarcho** [comment]
Those Indicators are the first child of ```<carousel>``` for me it was giving a blank Slide because .
```
if (this.index === 0) {
                this.$el.classList.add('active')
            }
```
so making 
```
if (this.index === 1) {
```
fixed the blank slide
I dont know if this is the correct way to do it.

**Comment by wffranco** [comment]
Carousel corrected in the last version.
